kml/edit-kml-ground-overlay/README.md

```diff
@@ -0,0 +1,39 @@
+# Edit KML ground overlay
+
+Edit the values of a KML ground overlay.
+
+![Edit KML ground overlay](EditKMLGroundOverlay.png)
+
+## Use case
+
+KML ground overlays are used for showing aerial imagery, symbology, or other images draped over a scene. Changing the geometry, rotation, and other attributes of a ground overlay after it has been loaded allows for live editing.  For example, editing the geometry and opacity of a historical image draped over present day satellite imagery makes it possible to view change over time.
+
+## How to use the sample
+
+Use the slider to adjust the opacity of the ground overlay.
+
+## How it works
+
+1. Create an `Envelope` defining the geometry of the overlay.
+2. Create a `KmlIcon` using a `Uri` linking to an image.
+3. Create a `KmlGroundOverlay` using the envelope and icon.
+4. Use `kmlGroundOverlay.setRotation()` to set the rotation of the ground overlay.
+5. Create a `KmlDataset` using the ground overlay.
+6. Create a `KmlLayer` using the dataset.
+7. Add the KML layer to the scene.
+8. Listen for changes to the opacity slider and use `kmlGroundOverlay.setColor(value)` to change the value appropriately.
```

Comment:
In other readmes, we've avoided using sample-specific elements in this section. Can we just say "Use `kmlGroundOverlay.setColor(value)` with a different alpha value to change the opacity."?